corehq/apps/app_manager/detail_screen.py

```diff
@@ -568,7 +572,10 @@ def xpath(self):
         property = parts.pop()
         indexes = parts
 
-        use_relative = property != '#owner_name'
+        if property == '#owner_name' or self.is_case_list_column:
```

Comment:
could you not just say `use_relative = bool(indexes) or property != '#owner_name'`